### PR TITLE
feat: Solana guide

### PR DESCRIPTION
**Pull request by mohandast52**
[Preview URL here](https://autonolas-bond-frontend-git-mohan-solana-guide-autonolas.vercel.app/paths/wsol-olas-via-orca-on-solana)

### CODE REVIEW

File: components/PathContent.jsx [hunk 1]
```diff
@@ -8,10 +8,29 @@ const StyledLink = styled.a`
   color: ${COLOR.GREY_2};
 `;
 
+const SvmDeposit = () => (
+  <>
+    Deposit SOL or WSOL and OLAS tokens using the Lockbox via
+    {' '}
+    <StyledLink href="https://tokenomics.olas.network/manage-solana-products" target="_blank" rel="noopener noreferrer">
```
Comment (Tanya-atatakai) [inline components/PathContent.jsx:15]: is this link correct? I see 404

File: components/Address.jsx [hunk 2]
```diff
@@ -32,9 +32,11 @@ const Address = ({ address, networkId }) => {
 Address.propTypes = {
   address: PropTypes.string.isRequired,
   networkId: PropTypes.string,
+  customExplorerUrl: PropTypes.string,
 };
 
 Address.defaultProps = {
   networkId: 'ethereum',
+  customExplorerUrl: undefined,
```
Comment (truemiller) [inline components/Address.jsx:40]: no need for this, customExplorerUrl will be undefined if it's not set by someone

File: components/PathContent.jsx (hunk 1 above)
Comment (truemiller) [reply components/PathContent.jsx:15]: yeah its in: https://github.com/valory-xyz/autonolas-tokenomics-frontend/pull/116

Comment (mohandast52) [reply components/PathContent.jsx:15]: yes, this has not yet been released

**Review by truemiller — APPROVED** [review]
looks good, tiny comment

**Review by Tanya-atatakai — APPROVED** [review]

**Review by mariapiamo — COMMENTED** [review]
- Change title and subtitle using the one suggested in the doc
- replace OLAS-WSOL with WSOL-OLAS everywhere as done in the guide doc

**Comment by mariapiamo** [comment]
6. Bridge WSOL-OLAS tokens from Solana to Ethereum using [Wormhole: Portal Token Bridge](https://portalbridge.com/advanced-tools/#/transfer).

should be replaced with 


6. Bridge WSOL-OLAS LP **fungible** tokens from Solana to Ethereum using [Wormhole: Portal Token Bridge](https://portalbridge.com/advanced-tools/#/transfer).

**Review by mariapiamo — APPROVED** [review]

**Comment by adrianoVal** [comment]
"To get fungible tokens representing [full range WSOL-OLAS LP positions on Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3) WSOL-OLAS pool. " Should this not link to this address CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR @mariapiamo ?

**Comment by adrianoVal** [comment]
"To get fungible tokens representing [full range WSOL-OLAS LP positions on Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3) WSOL-OLAS pool. " Should this not link to this address CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR @mariapiamo ?

**Comment by adrianoVal** [comment]
"You can acquire it on [Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3) directly on Solana" should link to https://www.orca.so/ instead

**Comment by mariapiamo** [comment]
> "To get fungible tokens representing [full range WSOL-OLAS LP positions on Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3) WSOL-OLAS pool. " Should this not link to this address CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR @mariapiamo ?

@adrianoVal no, that is correct. If you want to have a link to the fungible token, we should do like this:

"To get [fungible tokens](https://solscan.io/token/CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR) representing [full range WSOL-OLAS LP positions on Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3) WSOL-OLAS pool."

**Review by adrianoVal — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**